Agents/InequalityIndexCalculationAgent/python_scripts/README.md

```diff
@@ -0,0 +1,92 @@
+# Description
+
+The code and data collected in this folder is dedicated to reproduce the figures in **[Preprint 323](https://como.ceb.cam.ac.uk/preprints/323/): Impact of heat pumps and future energy prices on regional inequalities.**
+
+## Data
+Please download the data (there should be two folders called `Data` and `GB_shapefile` respectively) from the project folder in the the archive for preprint 323 (available on request from the corresponding author) and use it to replace the contents of the `./Data` and `./GB_shapefile` fiolders.
+
+## Project structure
+Once the data has been sorted, the project should be organised as follows:
+```bash
+code/ # The code to calculate, and generate the figures
+├── datamodel/
+│   ├── __init__.py
+│   └── functionality.py # Utility functions
+├── dataoutput/
+│   ├── __init__.py
+│   ├── dataoutput.py # Calculation functions and core figure generation functions
+│   └── Figure_generation.py # KEY script, you will need to run this script to reproduce the figure. 
+├── dataretrieval/
+│   ├── __init__.py
+│   └── dataretrieval.py # Functions to retrieve the data.
+└── __init__.py
+Data/ # Data used in this project
+│   ├── pickles/
+│   │     └── geometry # Geometry shapes of LSOA boundaries
+│   ├── properties_csv/ # Data for each lsoa or for the UK, separated by year
```

Comment:
What does "for each lsoa or for the UK" mean - could you be more precise?

For example, doe the boundaries have to be LSOAs or could they be any other type of region? Could the LSOAs (or other geometries) be for a part of the UK or the whole of the UK?

Reply:
I think it is  a good point here -- some comments may seem precise for me but confusing for reviewer, and it should be for the reviewer to judge -- let me know if there is other place that seems confusing for you,

Reply:
I am now happy with this.